.travis.yml

```diff
@@ -1,12 +0,0 @@
-language: ruby
-rvm:
-- 2.2.5
-
-install: gem install jekyll
-
-script: jekyll build
-
-env:
-  global:  # Ignore YAMLLintBear because indentation required
-  - NOKOGIRI_USE_SYSTEM_LIBRARIES=true # Ignore YAMLLintBear
-sudo: false
```

Comment:
travis.yml was required because of jekyll, but in case of angular we are not building anything right, that's why removed the travis.yml